hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -290,70 +287,77 @@ By default, no extra debug messages are enabled.
+### eeprom_write_protect  
+Controls whether the bootloader and VLI EEPROMs are marked as write protected.
 
-```
-# Update the rpi-eeprom package                                                                                                 
-sudo apt update
-sudo apt upgrade
-# Check the current version
-sudo rpi-eeprom-update     
-# Update to latest
-sudo rpi-eeprom-update -a
-```
+** This has no effect unless the EEPROM `/WP` pin is pulled low (TP5). Similarly, /WP low only write protects the status register so if EEPROM write protect was not previously defined then the EEPROM would not actually be write protected **
 
-### Enable network boot
-Network boot is not enabled by default in the bootloader. To enable it the bootloader configuration file must be edited.
-```                                                                        
-# Extract the configuration file                                                                                                         
-cp /lib/firmware/raspberrypi/bootloader/critical/pieeprom-2020-04-16.bin pieeprom.bin                                                       
-rpi-eeprom-config pieeprom.bin > bootconf.txt                                                                          
-```
-Change `BOOT_ORDER` to be `0x21` instead of `0x1`. This tells the bootloader to try sd-card boot first and network boot second. You should normally include sd-card (`0x1`) in the boot sequence in-case of network failure.
+See: [Winbond W25x40cl datasheet](https://www.winbond.com/resource-files/w25x40cl_f%2020140325.pdf)
+
+* 1 - Configures the write protect regions to cover the entire EEPROM.
+* 0 - Clears write protect regions
+* -1 - Do nothing.
+
+Default: -1  
+Version: 2020-09-03   
+
+### bootloader_update
+This option may be set to 0 to block self-update without requiring the EEPROM configuration to be updated. This is sometimes useful when updating multiple Pis via network boot because this option can be controlled per Raspberry Pi (e.g. via a serial number filter in config.txt).
 
-### Apply the configuration change to the EEPROM image file
+Default: 1
+Version: 2020-09-03  
+
+## Advanced boot modes - Network / USB mass storage boot.
+For network or USB mass storage boot we recommend updating to bootloader version 2020-09-03 or newer.
+
+### Updating the bootloader
+#### Update using the Raspberry Pi Imager
+The easiest method to update the bootloader with a factory default configuration supporting USB boot is to use the Raspberry Pi Imager.
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)
+1. Download the latest [EEPROM recovery image](https://github.com/raspberrypi/rpi-eeprom/blob/master/releases.md)
+1. Select `Use Custom` to reformat and flash a **blank** SD card with the EEPROM update enabling USB MSD boot support.
```

Comment:
Is the "Use Custom" step still needed, now that 2020-09-03 has been promoted to the critical release?